packages/react/src/OneSchemaImporter.tsx

```diff
@@ -42,36 +42,39 @@ export default function OneSchemaImporter(props: OneSchemaImporterProps) {
 
   useEffect(() => {
     return () => {
-      importer.close(true)
+      importer && importer.close(true)
     }
-  }, [])
+  }, [importer])
```

Comment:
I'm not sure all these if checks are necessary, but with how much react talks about async stuff, maybe it isn't safe to assume the useState is init'd first ??

I checked to make sure cleanup is still ran with this in deps..